python-client/giskard/core/suite.py

```diff
@@ -85,9 +109,7 @@ class SuiteInput:
     name: str
 
     def __init__(self, name: str, ptype: Any) -> None:
-        assert (
-            ptype in suite_input_types
-        ), f"Type should be one of those: {suite_input_types}"
+        assert ptype in suite_input_types, f"Type should be one of those: {suite_input_types}"
```

Comment:
"Type should be one of: xxx, yyy, zzz"
or "one of th*e*se"

(not related to this PR, but just noticed it)